pysat/tests/test_instruments.py

```diff
@@ -211,3 +219,42 @@ def test_load(self, inst, clean_level):
                 raise merr
         else:
             pytest.skip("Download data not available")
+
+    @pytest.mark.parametrize("inst", instruments['download'])
+    def test_remote_file_list(self, inst):
+        print(' '.join(('\nTrying:', inst.platform,
+                        inst.name, inst.tag, inst.sat_id)))
+        name = '_'.join((inst.platform, inst.name))
+        if hasattr(getattr(pysat.instruments, name), 'list_remote_files'):
+            assert callable(inst.remote_file_list)
+        else:
+            pytest.skip("remote_file_list not available")
+
+
+class TestInstrumentsNoDownload():
+    """Provides additional tests for instruments where downloads are not
+    currently available.  May be used for models that are locally run,
+    instruments not yet publically available, or instruments that require a
+    download interface not yet supported by pysat.
+
+    """
+
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
+        self.package = 'pysat.instruments'
+
+    def teardown(self):
+        """Runs after every method to clean up previous testing."""
+        del self.package
+
+    @pytest.mark.parametrize("inst", instruments['no_download'])
+    def test_download_warning(self, inst):
+        print(' '.join(('\nChecking download routine warnings for module: ',
```

Comment:
My guess is this print is still needed in case things don't work out?

Reply:
D'oh.  Wrote this bit before the new paradigm.  I'll fix it to only print on error.

Reply:
no worries!